Gemfile.lock

```diff
@@ -663,7 +663,7 @@ GEM
       rubocop (~> 1.41)
     rubocop-factory_bot (2.25.1)
       rubocop (~> 1.41)
-    rubocop-rails (2.24.0)
+    rubocop-rails (2.24.1)
       activesupport (>= 4.2.0)
       rack (>= 1.1)
```

Comment:
I just assumed Rack wasn't related, but the dependency was even visible in the diff; I should have paid more attention!